backend/tests/v5/e2e/routes/teamspaces/projects/models/common/tickets.groups.test.js

```diff
@@ -85,25 +89,171 @@ const setupBasicData = async ({ users, teamspace, project, fed, con, template, t
 		}
 
 		model.ticket = { ...cloneDeep(ticket), _id: ticketRes._id };
+
+		model.notFound = { _id: ServiceHelper.generateUUIDString() };
 		/* eslint-enable no-param-reassign */
 	}));
 };
 
+const setupExtIDTicket = (container) => {
+	const viewName = ServiceHelper.generateRandomString();
+	const template = {
+		...ServiceHelper.generateTemplate(),
+		properties: [
+			{
+				name: viewName,
+				type: propTypes.VIEW,
+			},
+		],
+		modules: [],
+	};
+	const ticket = ServiceHelper.generateTicket(template);
+
+	const rev = ServiceHelper.generateRevisionEntry();
+	const revId = rev._id;
+
+	const rootNode = ServiceHelper.generateBasicNode('transformation', revId);
+
+	// FIXME: constant reference
```

Comment:
@ChristosTsiotsias  please go through the FIXMEs in this file and replace the hard coded naming with a constant from the previous review.